### PR TITLE
Add another upgrade note and fix value name

### PR DESCRIPTION
**Pull request by kfox1111**

### CODE REVIEW

File: charts/spire/README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ We only support upgrading one major version at a time. Version skipping isn't su
 
 ### 0.17.X
 
+- If you set spire-server.replicaCount > 1, update it to 1 before upgrading and after upgrade you can set it back to its previous value.
 - The SPIFFE OIDC Discovery Provider now has many new TLS options and defaults to using SPIRE to issue its certificate.
 - The `spiffe-oidc-discovery-provider.insecureScheme.enabled` flag was removed. If you previously set that flag, remove the setting from your values.yaml and see if the new default of using a SPIRE issued certificate is suitable for your deployment. If it isn't, please consider one of the other options under `spiffe-oidc-discovery-provider.tls`. If all other options are still unsuitable, you can still enable the previous mode by disabling TLS. (`spiffe-oidc-discovery-provider.spire.enabled=false`)
```
Comment (marcofranssen) [inline charts/spire/README.md:75]: This needs a tiny fix.

```diff
- spiffe-oidc-discovery-provider.spire.enabled=false
+ spiffe-oidc-discovery-provider.tls.spire.enabled=false
```

Comment (kfox1111) [reply charts/spire/README.md:75]: doh. good catch. Will fix.

Comment (kfox1111) [inline charts/spire/README.md:75]:
```suggestion
- The `spiffe-oidc-discovery-provider.insecureScheme.enabled` flag was removed. If you previously set that flag, remove the setting from your values.yaml and see if the new default of using a SPIRE issued certificate is suitable for your deployment. If it isn't, please consider one of the other options under `spiffe-oidc-discovery-provider.tls`. If all other options are still unsuitable, you can still enable the previous mode by disabling TLS. (`spiffe-oidc-discovery-provider.tls.spire.enabled=false`)
```

**Review by marcofranssen — APPROVED** [review]
LGTM 🚀

**Comment by kfox1111** [comment]
tests seem to be stuck..... going to close/reopen to see if it will try again....

**Pull request merged**